src/lib/shopping-analytics.js

```diff
@@ -66,6 +66,7 @@ export const shoppingAnalyticsSetup = (config? : Config = {}) => {
     send: sendOrEnqueue,
     set: shoppingTracker.set,
     autoGenerateProductPayload: shoppingTracker.autoGenerateProductPayload,
+    getPageSkuData: capturePageData,
```

Comment:
Should this method somehow indicate it isn't for public use ? 
Eg: `__getPageSkuData` or `private__getPageSkuData` ? 

The idea is the capturing of SKU data would be transparent to the end user and will be controlled by a config.

Reply:
Is any other methods except `set` and `send` designed for public use?

Reply:
I think we can also remove `autoGenerateProductPayload` .... it isn't used on live .... @dturgumbaev - can you confirm ?